ha/util/message_bus.py

```diff
@@ -53,78 +54,61 @@ def publish(self, message: any):
         else:
             raise Exception(f"Invalid type of message {message}")
 
-class MessageBusConsumer:
-
-    JSON_TYPE = "json"
-    STRING_TYPE = "string"
+class MessageBusConsumer(Thread):
 
-    def __init__(self, consumer_id: int, consumer_group: str, message_types: list,
-                auto_ack: bool, offset: str):
+    def __init__(self, consumer_id: int, consumer_group: str, message_type: str,
+                callback: Callable, auto_ack: bool, offset: str):
         """
         Initalize consumer.
 
         Args:
             consumer_id (int): Consumer ID.
             consumer_group (str): Consumer Group.
             message_type (list): Message Type.
+            callback (Callable): function to get message.
```

Comment:
*to process message

Reply:
i will address it after rebase with
https://github.com/Seagate/cortx-ha/pull/542

Reply:
Done